lib/manager/tool-versions/extract.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { Fixtures } from '../../../test/fixtures';
+import { extractPackageFile } from './extract';
+
+describe('manager/tool-versions/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('returns null for empty', () => {
+      expect(extractPackageFile('', '.tool-versions', {})).toBeNull();
+    });
+
+    it('extracts dep versions for supported tools', () => {
+      const res = extractPackageFile(
+        Fixtures.get('.tool-versions'),
+        'unused_file_name',
+        {}
+      );
+      expect(res).toEqual({
+        deps: [
+          {
+            currentValue: '10.15.0',
+            currentVersion: '10.15.0',
+            datasource: 'github-tags',
+            depName: 'nodejs',
+            lookupName: 'nodejs/node',
+          },
+          {
+            currentValue: '3.2.5',
+            currentVersion: '3.2.5',
+            datasource: 'ruby-version',
+            depName: 'ruby',
+          },
+          {
+            currentValue: '2.1.3',
+            currentVersion: '2.1.3',
+            datasource: 'github-tags',
+            depName: 'python',
+            lookupName: 'python/cpython',
+          },
+        ],
+      });
+    });
+
+    it('extracts dep digests for supported tools', () => {
+      const res = extractPackageFile(
+        Fixtures.get('.tool-versions-refs'),
+        'unused_file_name',
+        {}
+      );
+      expect(res).toEqual({
+        deps: [
+          {
+            currentDigest: '10.15.0',
+            currentRawValue: 'ref:10.15.0',
```

Comment:
I'm not sure if this is what we like to have. 🤔

Reply:
Also, would `currentRawValue` be useful here or I should remove it?

Reply:
I think that if it's a version then this should be currentValue and not currentDigest, and we should use semver versioning